Sources/AppcuesKit/Presentation/Public/AppcuesPresentationDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppcuesPresentationDelegate.swift
+//  AppcuesKit
+//
+//  Created by Matt on 2024-03-04.
+//  Copyright © 2024 Appcues. All rights reserved.
+//
+
+import Foundation
+
+/// Metadata information for experiences in ``AppcuesPresentationDelegate``.
+@objc
+public class AppcuesPresentationMetadata: NSObject {
+    /// Appcues ID of the experience.
+    public let id: String
+
+    /// Name of the experience.
+    public let name: String
+
+    /// True if the experience is presented as an overlay. Includes modals, tooltips, and slideouts.
+    public let isOverlay: Bool
+
+    /// Context of the experience.
+    ///
+    /// Either `"modal"` or the `frameID` of an ``AppcuesFrame``.
+    public let renderContext: String
```

Comment:
no real harm to have both, little redundant? other idea would be to have `isEmbedded` and then an optional `frameID`... just thinking out loud about what the consumer might comprehend most easily